ManicDiggerLib/Server/Mods/War/War.cs

```diff
@@ -64,27 +120,46 @@ public enum SoldierSubclass
 			Rifle,
 		}
 		
-		TimeSpan RespawnTime = TimeSpan.FromSeconds(30);
-		//TimeSpan RoundTime = TimeSpan.FromMinutes(30);
+		TimeSpan RespawnTime = TimeSpan.FromSeconds(10);
 		DateTime CurrentRespawnTime;
 		
-		public bool EnableTeamkill = true;
+		public int currentRoundSeconds = 0;
+		public int secondsToEnd = 900;
```

Comment:
All these properties seem like it might be useful to introduce a class like `TeamStatistics` for them.
That would not only help keep things tidy but also allow for an arbitrary number of teams.